<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskCustomItem</class>
 <widget class="QWidget" name="TaskCustomItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame{
border: 1px solid rgb(0, 85, 0);
border-radius:5px;

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="thumbnaillabel">
        <property name="minimumSize">
         <size>
          <width>112</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>112</width>
          <height>63</height>
         </size>
        </property>
        <property name="text">
         <string>截图</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="name_label">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="durationlabel">
          <property name="text">
           <string>长度</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="info_label">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>info</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>37</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="state_label">
        <property name="text">
         <string>状态</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Relink_pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>重新链接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editpushButt">
        <property name="text">
         <string>编辑</string>
        </property>
        <property name="icon">
         <iconset resource="../Resource.qrc">
          <normaloff>:/new/prefix1/Resource/image/winOCRUI/slider.vertical.3.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/slider.vertical.3.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="state_image_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
